### PR TITLE
Multiline Field Support

### PR DESCRIPTION
**Pull request by csAyushDubey**
Ref: [VE-2556](https://contentstack.atlassian.net/browse/VE-2556)

Only change required on the CSR side is this CSS (on the multiline element): 
```css
white-space: pre-line
```

`dangerouslySetHTML` not required

[VE-2556]: https://contentstack.atlassian.net/browse/VE-2556?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Also Important:
- **Old state:** Bolds, italics or html allowed on multiline components and even get saved to CMS.
- **New state:** Bolds, italics or html show up just while editing contenteditable, **and change to plaintext as soon as user unfocusses**, only plaintext with newlines gets saved to CMS.

> Will have to take a separate ticket to block showing html styles while editing `contenteditable`.

### CODE REVIEW

File: src/liveEditor/utils/handleIndividualFields.ts [hunk 1]
```diff
@@ -259,3 +276,9 @@ export function cleanIndividualFieldResidual(elements: {
         focusedToolbar.innerHTML = "";
     }
 }
+
+const pasteAsPlainText = debounce((e: Event) => {
+    e.preventDefault();
+    const clipboardData = (e as ClipboardEvent).clipboardData;
```
[the remark this answers is not in the document]
Comment (csAyushDubey) [reply src/liveEditor/utils/handleIndividualFields.ts:282]: I was able to check on Chrome, Brave, Firefox and Safari. It seems to be working as expected, consistent behaviour across.

File: src/liveEditor/generators/generateOverlay.tsx [hunk 2]
```diff
@@ -120,20 +125,31 @@ export function hideFocusOverlay(elements: HideOverlayParams): void {
                     (VisualEditor.VisualEditorGlobalState.value
                         .previousSelectedEditableDOM as HTMLElement);
 
-                liveEditorPostMessage?.send(
-                    LiveEditorPostMessageEvents.UPDATE_FIELD,
-                    {
-                        data:
-                            "innerText" in actualEditedElement
+                let data = "innerText" in actualEditedElement
```
[the remark this answers is not in the document]
Comment (csAyushDubey) [reply src/liveEditor/generators/generateOverlay.tsx:128]: as discussed on slack

File: src/liveEditor/utils/getMultilinePlaintext.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+
+export function getMultilinePlaintext(editableElement: Element): string {
+  let newValue = '';
+  let isOnFreshLine = true;
+
+  // Recursive function to navigate childNodes and build linebreaks with text
+  function parseChildNodesForValueAndLines(childNodes: NodeListOf<ChildNode>) {
+      for (let i = 0; i < childNodes.length; i++) {
+          console.log(newValue);
```
[the remark this answers is not in the document]
Comment (csAyushDubey) [reply src/liveEditor/utils/getMultilinePlaintext.ts]: done

**Review by hiteshshetty-dev — APPROVED** [review]
LGTM!!

**Pull request closed without merging**